core/src/main/java/org/infinispan/util/logging/events/Messages.java

```diff
@@ -71,4 +71,10 @@
 
    @Message(value = "After merge (or coordinator change), the coordinator failed to recover cluster. Cluster members are %s.")
    String clusterRecoveryFailed(Collection<Address> members);
+
+   @Message(value = "Site '%s' is online.", id = 100004)
```

Comment:
Where does this id come from? Last messages have no id. Others have ids in a completely different range. Maybe @danberindei or @tristantarrant know?